src/pages/webhooks/create-webhooks.md

```diff
@@ -65,55 +97,92 @@ Field | Description
 
 ### Configure hook fields
 
-The **Hook Fields** configuration panel defines the payload of a webhook request. [Define the hook body](./hooks.md#define-the-hook-body) describes how to construct the payload.
+The **Hook Fields** configuration panel or the `fields` element in the `webhooks.xml` file defines the payload of a webhook request and maps the fields of the webhook method to the payload structure required by the remote server.
 
-Field | Description
---- | ---
-**Name** | The path to the field to include in the transmitted webhook, such as `product.sku`.
-**Source** | The path to the value in the default webhook. If not set, the **Name** value is used.
-**Active** | Indicates whether to include the field in the payload.
+Admin field | XML attribute | Description
+--- | --- | ---
+| **Name** | `field.name`| The path to the field to include in the transmitted webhook, such as `product.sku`. |
+| **Source** | `field.source`| The path to the value in the default webhook, such as `quoteItem.sku`. If not set, the **Name** value is used. |
+| **Active** | `field.remove` | Indicates whether to include the field in the payload. By default, all fields are included. If you are building a `webhooks.xml` file, set `field.remove` to `true` to remove the field from the payload. |
+| - | `field.converter` | A class that transforms the value of a field, such as from integer to string. This attribute is only available in `webhooks.xml` files. |
 
-### Configure hook headers
+[Define hooks](./hooks.md) describes how to define the fields of a webhook request.
 
-The **Hook Headers** configuration panel defines the headers of a webhook request. [Define request headers](./hooks.md#define-request-headers) describes how to send authorization tokens and other connection parameters.
+### Configure hook headers
 
-Field | Description
---- | ---
-**Name** | The header name, in the same form as it will be sent. For example, `Authorization`.
-**Value** | The value of the header, such as `Bearer: <token>`.
-**Active** | Set to **No** to remove the header from the request.
+<InlineAlert variant="info" slots="text"/>
+
+In PaaS environments, you define authorization tokens and other connection parameters in the `headers` element of a `webhooks.xml` file. In SaaS environments, you define similar authentication information in the **Developer Console OAuth** configuration panel. However, you can still define additional headers in the **Hook Headers** configuration panel.
```

Comment:
The `Developer Console OAuth` feature was built with sending requests to secured Adobe I/O runtime actions in mind. Not sure if that's worth mentioning

Reply:
I'll leave this as is.